src/main/java/de/tum/in/www1/artemis/service/connectors/bamboo/BambooService.java

```diff
@@ -692,6 +692,7 @@ private void addFeedbackToResult(Result result, List<BambooBuildResultNotificati
                 artifactLabelFilter.add("Build log");
                 buildResult.getArtifacts().setArtifacts(
                         buildResult.getArtifacts().getArtifacts().stream().filter(artifact -> !artifactLabelFilter.contains(artifact.getName())).collect(Collectors.toList()));
+                buildResult.getArtifacts().setSize(buildResult.getArtifacts().getArtifacts().size());
```

Comment:
Not directly related to this PR, but we should maybe remove this redundant size attribute and just use the size of the list (as this is only a DTO, the list should always be present, AFAIK).